.npmrc

```diff
@@ -1 +1 @@
-//registry.npmjs.org/:_authToken=${NODE_AUTH_TOKEN}
\ No newline at end of file
+#//registry.npmjs.org/:_authToken=${NODE_AUTH_TOKEN}
```

Comment:
Why is this commented out?

Reply:
This is causing issues in builds. We don't need this to be enabled currently.